src/components/Claudy.svelte

```diff
@@ -0,0 +1,57 @@
+<div class='{{`coz-claudy coz-bar-hide-sm ${opened ? "coz-claudy--opened" : ""}`}}'>
+  <button class='coz-claudy-icon' data-claudy-opened='{{opened}}' on:click='toggleClaudy()'/>
+  <ClaudyMenu actions='{{config.actions}}' on:close='toggleClaudy()' usageTracker='{{usageTracker}}' />
+</div>
+
+<script>
+  /* global __PIWIK_TRACKER_URL__  __PIWIK_SITEID__ __PIWIK_DIMENSION_ID_APP__*/
+  import { t } from '../lib/i18n'
+
+  import ClaudyMenu from './ClaudyMenu'
+  import { shouldEnableTracking, getTracker, configureTracker } from '../lib/piwik'
+
+  export default {
+    data () {
+      return {
+        opened: false,
+        usageTracker: null
+      }
+    },
+
+    oncreate () {
+      // if tracking enabled, init the tracker
+      if (shouldEnableTracking()) {
+        const trackerInstance = getTracker(__PIWIK_TRACKER_URL__, __PIWIK_SITEID__, false, false)
+        configureTracker({
+          appDimensionId: __PIWIK_DIMENSION_ID_APP__,
+          app: 'Cozy Bar',
```

Comment:
Maybe you should check with @jsilvestre under what app name Claudy visits should appear. It's quite possible this should use the name of the actual app and not Cozy Bar, but idk.